ocaml/xe-cli/bash-completion

```diff
@@ -140,8 +142,6 @@ _xe()
                         cmd=vm-list;;
                     diagnostic-vdi-status)
                         cmd=vdi-list;;
-                    host-cpu-info)
-                        cmd=host-list;;
```

Comment:
Was this removed on purpose? It's not mentioned in the commit message

Reply:
this case is covered by the wildcard host-cpu-* just below

Reply:
Oops, not really. I'll revert this particular change.